src/ruby/yast/wfm.rb

```diff
@@ -331,6 +331,9 @@ def self.run_client(client)
         handle_abort_exception(e, client)
         exit
       rescue Exception => e
+        # Don't interfere with RSpec, such as RSpec::Mocks::MockExpectationError
+        raise e if e.class.to_s.start_with?("RSpec::")
```

Comment:
@jreidinger this is the fix mentioned in https://github.com/yast/yast-yast2/pull/1004

Reply:
can be written as `raise if e.class.to_s.start_with?("RSpec::")`